src/connector/src/parser/avro/util.rs

```diff
@@ -15,20 +15,28 @@
 use std::sync::LazyLock;
 
 use apache_avro::schema::{DecimalSchema, RecordSchema, Schema};
+use apache_avro::types::{Value, ValueKind};
 use itertools::Itertools;
 use risingwave_common::bail;
 use risingwave_common::log::LogSuppresser;
 use risingwave_common::types::{DataType, Decimal};
 use risingwave_pb::plan_common::{AdditionalColumn, ColumnDesc, ColumnDescVersion};
 
 use crate::error::ConnectorResult;
+use crate::parser::unified::bail_uncategorized;
+use crate::parser::{AccessError, MapHandling};
 
-pub fn avro_schema_to_column_descs(schema: &Schema) -> ConnectorResult<Vec<ColumnDesc>> {
+pub fn avro_schema_to_column_descs(
+    schema: &Schema,
+    map_handling: Option<MapHandling>,
```

Comment:
I think this should be `MapHandling` instead of `Option`. If user didn't specify, the default value will be used when handling the parameters, not here.

Reply:
default behavior is reject

Reply:
Is it because `Map` is not supported yet? 

Oh, I see. I suppose that after `Map` is supported, `Map` will be the default value here, right?

Reply:
Yes 👍